app/src/features/mocks/contexts/actions.tsx

```diff
@@ -17,18 +18,23 @@ import { toast } from "utils/Toast";
 import { isRecordMock } from "../screens/mocksList/components/MocksList/components/MocksTable/utils";
 import { updateMocksCollectionId } from "backend/mocks/updateMocksCollectionId";
 import { DEFAULT_COLLECTION_ID } from "../constants";
+import { getFunctions, httpsCallable } from "firebase/functions";
+import * as FilesService from "../../../utils/files/FilesService";
+import { deleteMock } from "backend/mocks/deleteMock";
 
 type MocksActionContextType = {
   createNewCollectionAction: (mockType: MockType) => void;
   updateCollectionNameAction: (mockType: MockType, record: RQMockMetadataSchema) => void;
   deleteCollectionModalAction: (record: RQMockMetadataSchema) => void;
-  deleteMockModalAction: (record: RQMockMetadataSchema) => void;
-  updateMockCollectionModalAction: (record: RQMockMetadataSchema) => void;
+  deleteRecordsModalAction: (records: RQMockMetadataSchema[], onSuccess?: () => void) => void;
```

Comment:
Rename this to `deleteRecordsAction`. 
This is normal delete actions which internally triggers deleteRecordsModalAction

Reply:
- deleteCollectionModalAction -> deleteCollectionAction
- updateMocksCollectionModalAction -> updateMocksCollectionAction
- mockUploaderModalAction -> uploadMockAction
- newFileModalAction -> createNewFileAction
- createNewMockAction -> createNewMockAction
-

Reply:
We already have `deleteRecordsAction`